charts/argo-workflows/values.yaml

```diff
@@ -706,15 +706,16 @@ artifactRepository:
   # -- Store artifact in a S3-compliant object store
   # @default -- See [values.yaml]
   s3:
+    secretName: "{{ .Release.Name }}-minio"
     # Note the `key` attribute is not the actual secret, it's the PATH to
     # the contents in the associated secret, as defined by the `name` attribute.
     accessKeySecret:
-      name: "{{ .Release.Name }}-minio"
```

Comment:
I am confused now. This is supposed to be a default, and "{{ .Release.Name }}-minio"` is a perfectly good one. Why change it to a template expansion here, especially a recursive one? It's already expanded at the [call site](https://github.com/argoproj/argo-helm/blob/7959fee689ec3a72df5c1d966044c3d93e8f414a/charts/argo-workflows/templates/controller/workflow-controller-config-map.yaml#L72).

Same for `secretKeySecret`.